configuration/audit-log-for-settings.md

```diff
@@ -0,0 +1,187 @@
+---
+description:  This article describes how you can access the audit log for all changes made in the Settings of Syskit Point. 
+---
+
+# Audit for Settings
+
```

Comment:
add info that this article describes the process of accessing settings audit logs for Data Center customers. SaaS customers should contact support if they need these logs.